### ISSUE TITLE
Wrong results handling wikilink links with custom display text

### ISSUE THREAD

**Issue by tuxpeople** [symunona/obsidian-bulk-exporter#2]
Hi

Not sure if it's my fault or a bug, but I've got the following in a note I tried to export:

```
Die kontinuierliche Verbesserung ist ein elementarer Bestandteil [[agil|agiler]] Verfahren.
```

The name of the link is `agiler`, but it is pointing to `agil`. This type of link is a wikilink with custom display text, as described [here](https://help.obsidian.md/Linking+notes+and+files/Internal+links#Change+the+link+display+text). I get the following error while exporting:

<img width="238" alt="image" src="https://github.com/symunona/obsidian-bulk-exporter/assets/1506076/7941af85-16a6-4ace-9895-c54f1eb247a3">

It looks like it doesn't matter if the `agile` note exists. I think the export tries to remove the link (as expected), but what's left is not `agiler`, it's `agil|agiler`. The above example is shown as follows in the exported file:

```
Die kontinuierliche Verbesserung ist ein elementarer Bestandteil agil|agiler Verfahren.
```

**Comment by symunona** [comment]
Thank you very much for your feedback @tuxpeople! 

Indeed I did not parse `[[link|text]]` styled links properly, as I was not aware of the proper syntax.
I believe I fixed it in the latest release 2.0.2, please confirm!

**Comment by tuxpeople** [comment]
Wow, that was amazingly fast @symunona, thanks a lot! I just updated the plugin to version 2.0.2 and it's now able to export the above example correctly. So I'd say the issue is solved!
